Sources/Apollo/GraphQLResultAccumulator.swift

```diff
@@ -8,9 +8,9 @@ protocol GraphQLResultAccumulator: AnyObject {
   associatedtype ObjectResult
   associatedtype FinalResult
 
-  func accept(scalar: JSONValue, info: FieldExecutionInfo) throws -> PartialResult
-  func acceptNullValue(info: FieldExecutionInfo) throws -> PartialResult
-  func acceptMissingValue(info: FieldExecutionInfo) throws -> PartialResult
+  func accept(scalar: JSONValue, firstReceivedAt: Date, info: FieldExecutionInfo) throws -> PartialResult
+  func acceptNullValue(firstReceivedAt: Date, info: FieldExecutionInfo) throws -> PartialResult
+  func acceptMissingValue(firstReceivedAt: Date, info: FieldExecutionInfo) throws -> PartialResult
```

Comment:
❓ I don't remember why I marked them as `Date?` in the original changes - is that not necessary this time?

Reply:
if you are talking about `firstReceivedAt` parameter, for what I see it was never `Date?`

Reply:
If you look at the PR that fixed the min vs max age that I linked in another comment - I changed the values to `Date?`

I wonder if it was me refining the protocol, and giving a quirk of how protocol conformance works

Reply:
I think it was you refining the protocol

Reply:
Got it, never mind then